src/Annotation/GraphQLRateLimit.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bedrock\Bundle\RateLimitBundle\Annotation;
+
+use Bedrock\Bundle\RateLimitBundle\Model\GraphQLEndpointConfiguration;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @Annotation
+ * @Target({"METHOD"})
+ */
+final class GraphQLRateLimit
```

Comment:
Just a thought, but because this bundle is open sourced, making the class final might be frustrating for developers using it if they want to override / extend it 🤔 
I had this issue with the [gaufrette bundle](https://github.com/KnpLabs/KnpGaufretteBundle).

Reply:
I understand your concern and might be a good evolution, I did it the say way than the original annotation though.

Reply:
and in fact final is not so bad. Instead of using extend class use decoration it's call [composition over inheritence](https://stackoverflow.com/questions/49002/prefer-composition-over-inheritance)